### ISSUE TITLE
Add Leadership Team Section to Members Page

### ISSUE THREAD

**Issue by dominicduffin1** [Virtual-Coffee/virtualcoffee.io#64]
## Issue Context

Following the announcement of additions to the leadership team, I was thinking there should be a third section on the Members page headed 'Leadership Team'.

## Suggested Solution

Underneath the 'Our Founders' section, before the 'Our Members' section, I would add a 'Our Leadership Team' section.

## Alternatives Considered

Alternatively, there could be one 'Leadership' section with titles on the cards for the founders.

---------------

If this change is desired, I'd be interested in working on it.

**Comment by danieltott** [comment]
@dominicduffin1 Thank you for this suggestion! I think that's a great idea. We talked it over a bit and I think a good approach here would be something like this:

```
# Leadership Team
## Our Founders
Bekah | Sara
## Community Engagement
Bryan | Dan | Kirk
```

So there's one Leadership Team section with smaller subheaders for "Founders" and "Community Engagement"

I don't have a specific design in mind, so fair warning: this may take a bit of back and forth to nail down the design, just so you know what to expect 😄 

Feel free to message me on Slack or here if you have any questions or would like to pair up on this - but I'm excited to have your help with it!

**Comment by dominicduffin1** [comment]
@danieltott Great! I'm fine with some back and forth, will let you know when I have a first go ready to show you, or if I need any help getting there!

**Comment by danieltott** [comment]
@dominicduffin1 Awesome! Feel free to open a PR when you're ready to go over some stuff - if you just put "Work in progress" or something like that in the title we'll know not to merge it right away. That way we'd have a Deploy Preview to work with and all of that.

**Comment by danieltott** [comment]
Closed via #68
Thank you @dominicduffin1!!

![mission accomplished](https://media0.giphy.com/media/UTjccaRoigEJx2DKnk/giphy.gif)
